api/users.js

```diff
@@ -0,0 +1,92 @@
+var express = require('express');
+var router = express.Router();
+const db = require('../db/db');
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt'); // pass ne smije biti NUL
+
+// GET
+router.get('/', (req, res) => db.users.findAll({attributes: {exclude: ['password']}}).then(users => res.json(users)));
+
+router.get('/:id', (req, res) => db.users.findOne({
+    where: {   id: req.params.id } , attributes: {exclude: ['password']}}).then( data => { res.send(data) })   
+);
+
+// DELETE
+router.delete('/:id' , (req, res) => db.users.destroy({
+    where: {   id: req.params.id     }
+ }).then( () => { res.json({ status : 'Deleted!'}) })  
+);
+
+
+router.get('/profile', (req, res) => {
+    var decoded = jwt.verify(req.headers['authorization'], process.env.SECRET_KEY)
+    
+    db.users.findOne({
+        where: {
+            id: decoded.id
+        }
+    }).then(user => {
+        if (user) {
+            res.json(user)
+        } else {
+            res.send('User does not exist')
+        }
+        })
+        .catch(err => {
+        res.send('error: ' + err)
+        })
+});
+
+// POST
+router.post('/login', (req, res) => {
+    db.users.findOne({
+        where: {
+            email: req.body.email,
+            password: req.body.password
+        }
+    })
+    .then(user => {
+    if (bcrypt.compareSync(req.body.password,user.password) || req.body.password == user.password ) {
```

Comment:
Empty space should always be after a comma `bcrypt.compareSync(req.body.password, user.password)`

Reply:
=== instead of ==